ProjectName/src/navigation/navtabs.styles.ts

```diff
@@ -10,6 +10,12 @@ const navTabsStyle = StyleSheet.create({
     fontSize: 18,
     fontWeight: '500',
   },
+  PaymentStyle: {
```

Comment:
I would suggest omit `style` prefix, since this entity already exists inside `navTabsStyle`. So it's better to name it `payment` or `paymentBlock` or something related.